public/app/plugins/datasource/alertmanager/ConfigEditor.tsx

```diff
@@ -33,6 +33,17 @@ const IMPL_OPTIONS: Array<SelectableValue<AlertManagerImplementation>> = [
 export const ConfigEditor = (props: Props) => {
   const { options, onOptionsChange } = props;
 
+  // As we default to Mimir, we need to make sure the implementation is set from the start
+  useEffect(() => {
```

Comment:
I wonder if there is no better place somewhere in `public/app/plugins/datasource/alertmanager/DataSource.ts` Alertmanager data source file for this type of initialization, but I'm not very familiar with data sources

Reply:
yep, I was wondering the same question and I didn't find another way to do it